buildSrc/src/main/kotlin/SqlDelightTmdbCacheDefinitionsGenerator.kt

```diff
@@ -0,0 +1,110 @@
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.TaskAction
+import java.io.File
+
+/**
+ * A Gradle Task to generate the SQLDelight definitions related to the Tmdb Cache.
+ */
+open class SqlDelightTmdbCacheDefinitionsGenerator : DefaultTask() {
+    /** Where to generate the .sq files */
+    private val sqldelightRoot = File(project.projectDir, "src/androidMain/sqldelight/tmdbCache/io/github/couchtracker/db/tmdbCache")
```

Comment:
Can we use the Path API instead? 

You can also do fun stuff like:
```kotlin
private val sqldelightRoot = project.projectDir.toPath() / "src" / "androidMain" / "sqldelight" / "tmdbCache" / "io" / "github" / "couchtracker" / "db" / "tmdbCache"
```

Reply:
Why?